### PR TITLE
add(mp): Azure Application Insights

### PR DESCRIPTION
**Pull request by itoussies**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [X] 20.04.x
- [X] 20.10.x
- [X] 21.04.x (master)

### CODE REVIEW

**Comment by itoussies** [comment]
WIP

File: en/integrations/plugin-packs/procedures/cloud-azure-management-applicationinsights.md [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+
+The Plugin is using a cache file to keep connection information and avoid an authentication at each call. 
+If some of the authentication parameters change, you must delete the cache file. 
+
+The cache file can be found within  ```/var/lib/centreon/centplugins/``` folder with a name similar to azure_api_<md5>_<md5>_<md5>_<md5>.
+
+### ```UNKNOWN: Login endpoint API returns error code 'ERROR_NAME' (add --debug option for detailed message)```
+
+When I run my command I obtain the following error message:
+```UNKNOWN: Login endpoint API returns error code 'ERROR_NAME' (add --debug option for detailed message)```.
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/cloud-azure-management-applicationinsights.md:370]: Maybe section title is enough nope?

File: fr/integrations/plugin-packs/procedures/cloud-azure-management-applicationinsights.md [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+Le Plugin utilise un fichier de cache pour conserver les informations de connexion afin de ne pas 
+se ré-authentifier à chaque appel. Si des informations sur le Tenant, la Souscription ou les 
+Client ID / Secret changent, il est nécessaire de supprimer le fichier de cache du Plugin. 
+
+Celui ci se trouve dans le répertoire ```/var/lib/centreon/centplugins/``` avec le nom azure_api_<md5>_<md5>_<md5>_<md5>.
+
+### ```UNKNOWN: Login endpoint API returns error code 'ERROR_NAME' (add --debug option for detailed message)```
+
+Lors du déploiement de mes contrôles, j'obtiens le message suivant : 
+```UNKNOWN: Login endpoint API returns error code 'ERROR_NAME' (add --debug option for detailed message)```.
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/cloud-azure-management-applicationinsights.md:372]: See EN version comment

File: en/integrations/plugin-packs/procedures/cloud-azure-management-applicationinsights.md (hunk 1 above)
Comment (itoussies) [reply en/integrations/plugin-packs/procedures/cloud-azure-management-applicationinsights.md]: Yes indeed

**Review by Sims24 — APPROVED** [review]

**Pull request merged**